### PR TITLE
Run rspec/inspec tests in Travis

### PR DESCRIPTION
**Pull request by metadave**
note, I cleanup up a single whitespace line in sup to trigger the build.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,9 +8,30 @@ env:
...
+      directories:
+      - /root/travis_bootstrap
+    before_install:
+    - ./support/ci/fast_pass.sh || exit 0
+    script:
+    - sudo ./support/ci/specs.sh
   - language: rust
     env:
-    - COMPONENTS=bin LIBSODIUM=$HOME/pkgs/libsodium/1.0.8 LIBARCHIVE=$HOME/pkgs/libarchive/3.2.0 PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$LIBARCHIVE/lib/pkgconfig:$LIBSODIUM/lib/pkgconfig" LD_LIBRARY_PATH="LD_LIBRARY_PATH:$LIBARCHIVE/lib:$LIBSODIUM/lib"
+    - COMPONENTS=bin LIBSODIUM=$HOME/pkgs/libsodium/1.0.8 LIBARCHIVE=$HOME/pkgs/libarchive/3.2.0 PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$LIBARCHIVE/lib/pkgconfig:$LIBSODIUM/lib/pkgconfig" LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$LIBARCHIVE/lib:$LIBSODIUM/lib"
```
Comment (reset) [inline .travis.yml]: The missing 💵 😦

**Comment by reset** [comment]
gif-keyboard-14456779782271188813

**Comment by reset** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/164765065).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/164765065).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**